### PR TITLE
schema change RDDM

### PR DESCRIPTION
**Pull request by nemi-notrace**

### CODE REVIEW

File: packages/repco-prisma/prisma/schema.prisma [hunk 1]
```diff
@@ -377,6 +377,18 @@ model Transcript {
   MediaAsset MediaAsset @relation(fields: [mediaAssetUid], references: [uid])
 }
 
+/// @repco(Entity)
+model Translation {
```
Comment (Frando) [inline packages/repco-prisma/prisma/schema.prisma:381]: Let's leave this out for now, it's not yet used in core and I think we should talk about internationalization a bit more before.

File: packages/repco-prisma/prisma/schema.prisma [hunk 2]
```diff
@@ -295,14 +295,14 @@ model Contribution {
...
-  Actors       Actor[]
+  Contributers Contributer[]
   ContentItems ContentItem[]
   MediaAssets  MediaAsset[]
   Revision     Revision      @relation(fields: [revisionId], references: [id])
 }
 
 /// @repco(Entity)
-model Actor {
+model Contributer {
```
Comment (Frando) [inline packages/repco-prisma/prisma/schema.prisma:305]: spelled with an `o`, so `Contributor`

**Review by Frando — COMMENTED** [review]
LGTM. Can you address the two comments and re-run the codegen? Will merge afterwards.

**Comment by nemi-notrace** [comment]
see [pr43](https://github.com/openaudiosearch/repco-specs/pull/43)

**Pull request closed without merging**